README.md

```diff
@@ -11,53 +11,44 @@ New Relic requires the following environment variables to identify Kubernetes ob
+- Only inject on pods created in specific namespaces, based on their labels.
 
-- creates `newrelic-metadata-injection-deployment` and `newrelic-metadata-injection-svc`.
-- registers the `newrelic-metadata-injection-svc` service as a MutatingAdmissionWebhook with the Kubernetes API.
+If you wish to use the label-based solution: download the yaml file, look for `namespaceSelector`, uncomment it and the 2 other lines right below (until `newrelic-metadata-injection: enabled`). There are comments in this part of the yaml to help you uncomment the correcty lines.
 
-### 3) Install the certificates
+This webhook needs to be authenticated by the Kubernetes extension API server, so it will need to have a signed certificate from a CA trusted by the extension API server. The certificate management is isolated from the webhook server and a secret is used to mount them. So there are 2 installation options:
 
-This webhook needs to be authenticated by the Kubernetes extension API server, so it will need to have a signed certificate from a CA trusted by the extension API server. The certificate management is isolated from the webhook server and a secret is used to mount them. 
+1. [With **automatic certificate** management](#automatic-certificate-management), recommended if you don't manually manage your certificates.
+2. [With **manual certificate** management](#manual-certificate-management), if you like to manage your certificate manually.
 
-**Important**: the webhook server has a file watcher pointed at the secret's folder that will trigger a certificate reload whenever anything is created or modified inside the secret. This allows easy certificate rotation with an update of the TLS secret that is created by running:
+Installing the injection, no matter which options chosen, will end up creating these resources in your Kubernetes cluster:
 
-```bash
-$ namespace=default # Change the namespace here if you also changed it in the yaml files.
-$ serverCert=$(kubectl get csr newrelic-metadata-injection-svc.${namespace} -o jsonpath='{.status.certificate}')
-$ tmpdir=$(mktemp -d)
-$ echo ${serverCert} | openssl base64 -d -A -out ${tmpdir}/server-cert.pem
-$ kubectl patch secret newrelic-metadata-injection-secret --type='json' \
-    -p "[{'op': 'replace', 'path':'/data/tls.crt', 'value':'$(serverCert)'}]"
-$ rm -rf $(tmpdir)
-```
+- `newrelic-metadata-injection-deployment` and `newrelic-metadata-injection-svc`.
+- `newrelic-metadata-injection-cfg`, registering the `newrelic-metadata-injection-svc` service as a MutatingAdmissionWebhook with the Kubernetes API.
 
-#### Automatic management
+### Automatic certificate management
 
 The certificate management can be automatic, using the Kubernetes extension API server (recommended, but optional):
 
 ```bash
-$ kubectl apply -f deploy/job.yaml
+$ kubectl apply -f http://download.newrelic.com/infrastructure_agent/integrations/k8s-metadata-injection-latest.yaml
```

Comment:
This is not correct. The user has to edit the file and update the value of the `clusterName` environment variable.